lib/get-latest-ticket-numbers.rb

```diff
@@ -1,39 +1,32 @@
 require 'zendesk_api'
 
-@client = ZendeskAPI::Client.new do |config|
-  config.url = ENV['ZENDESK_URL']
-  config.username = ENV['ZENDESK_USER_EMAIL']
-  config.password = ENV['ZENDESK_USER_PASSWORD']
-
-  config.retry = true
-end
+require_relative 'zendesk-setup.rb'
 
 # year-364 days means we will delete 1 xtra day and be compliant for 23:59:59h
 # but then we will run again in 24h time
 
-today = Date.today.prev_day
-lastyear = today - 365
+today = Date.today
+lastyear = Date.today.prev_day - 365
 
 @latest_tickets = []
 
 #
 puts "Total Tickets 2018 to 1 year ago today"
 #
 
-puts @client.search(:query => "type:ticket group_id:20188163 status:closed updated_at>2018-01-01 updated_at<#{lastyear}").count
+puts @client.search(:query => "type:ticket group_id:20188163 status:closed updated_at>2018-01-01 updated_at<#{lastyear}").count.to_i
```

Comment:
Can we update the query to include 1st January?

```suggestion
puts @client.search(:query => "type:ticket group_id:20188163 status:closed updated_at>=2018-01-01 updated_at<#{lastyear}").count.to_i
```

Reply:
Resolved as suggested